protected/humhub/libs/DynamicConfig.php

```diff
@@ -98,10 +98,10 @@ public static function rewrite()
             $config['language'] = Yii::$app->language;
         }
 
-        $defaultTimeZone = Yii::$app->settings->get('defaultTimeZone');
-        if (!empty($defaultTimeZone)) {
-            $config['timeZone'] = $defaultTimeZone;
-            $config['components']['formatter']['defaultTimeZone'] = $defaultTimeZone;
+        $serverTimeZone = Yii::$app->settings->get('serverTimeZone');
```

Comment:
@yurabakhtin The long-term goal is to eliminate DynamicConfig. 

It probably makes sense to remove the time zone change here as part of this refactoring.

Perhaps it makes sense to develop a “LocalizationService” that handles both time zones and language settings centrally and cleanly. Database + app.

Reply:
@luke- Ok, I have removed the options `timeZone` and `components / formatter / defaultTimeZone` from the dynamic config file.

Reply:
@yurabakhtin Can you please also "unset" these options, on the DynamicConfig save.